public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -100,208 +97,213 @@ code-example(language="sh" class="code-shell").
-    The list and the single hero are not connected in any way.
-    We want the user to select a hero from our list, and have the selected hero appear in the details view.
-    This UI pattern is widely known as "master-detail".
-    In our case, the master is the heroes list and the detail is the selected hero.
+    ## Selecting a hero
+    The app now displays a list of heroes as well as a single hero in the details view, but
+    the list and the details view are not connected.
+    When users select a hero from the list, the selected hero should appear in the details view.
+    This UI pattern is known as "master-detail."
+    <!-- CF: To be consistent with the title of this page, should this be "Master/Detail"? Or vice-versa? -->
+    In this case, the master is the heroes list and the detail is the selected hero.
 
-    Let’s connect the master to the detail through a `selectedHero` component property bound to a click event.
+    Next you'll connect the master to the detail through a `selectedHero` component property,
+    which is bound to a click event.
 
-    ### Click event
-    We modify the `<li>` by inserting an Angular event binding to its click event.
+    ### Add a click event
+    Modify the `<li>` by inserting an Angular event binding to its click event.
 
-  +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'selectedHero-click', 'app.component.ts (template excerpt)')
+  +makeExample('toh-2/ts/app/app.component.1.html', 'selectedHero-click', 'app.component.ts (template excerpt)')(format='.')
 
   :marked
-    Focus on the event binding
-  code-example.
+    Notice the event binding:
     (click)="onSelect(hero)"
   :marked
-    The parenthesis identify the `<li>` element’s  `click` event as the target.
-    The expression to the right of the equal sign calls the  `AppComponent` method, `onSelect()`,
+    The parentheses identify the `<li>` element's  `click` event as the target.
+    The `onSelect(hero)` expression calls the  `AppComponent` method, `onSelect()`,
     passing the template input variable `hero` as an argument.
-    That’s the same `hero` variable we defined previously in the `ngFor`.
+    That's the same `hero` variable you defined previously in the `ngFor` directive.
   .l-sub-section
     :marked
-      Learn more about Event Binding in the 
-      [User Input](../guide/user-input.html) and
-      [Templating Syntax](../guide/template-syntax.html#event-binding) chapters.
-  :marked
-    ### Add the click handler
-    Our event binding refers to an `onSelect` method that doesn’t exist yet.
-    We’ll add that method to our component now.
-
-    What should that method do? It should set the component’s selected hero to the hero that the user clicked.
+      Learn more about event binding at the 
+      [User Input](../guide/user-input.html) page and the
+      [Event binding](../guide/template-syntax.html#event-binding) section of the
```

Comment:
Is this a new style rule? I don't see anything about this in the [Angular Team Writing Guidelines](https://docs.google.com/document/d/1ETcX_38vkR8NkGyKqhWoCnPDg3WuLw8UifO5TGCHbSo/edit#heading=h.vvtblk5bmjf). I was using some examples in the glossary as a guide--see "Input" and "Interpolation" [here](https://angular.io/docs/ts/latest/glossary.html#!#I).